package.json

```diff
@@ -17,6 +24,10 @@
     "postlink-dist": "cd dist && rm -rf node_modules",
     "unlink-dist": "cd dist && npm unlink && rm package*",
     "watch": "NODE_ENV=development rollup --watch -c",
+    "i18n-scan": "i18next-scanner --config i18next-scanner.config.js src/**/*.{js,jsx}",
+    "i18n-conv-json": "i18next-conv -l en -s ./src/lib/translations/en/translations.json -t ./src/lib/translations/translations.pot",
+    "i18n-conv-po": "i18next-conv -l $npm_config_lang -s ./src/lib/translations/$npm_config_lang/messages.po -t ./src/lib/translations/$npm_config_lang/translations.json",
+    "i18n-conv-po-all": "node ./getTextConverter.js",
```

Comment:
@mb-wali what about merging the `i18n-conv-po` and this command by simply adding an extra argument e.g `--all` to comply also with what `tx` command is using for pulling all languages? I can approve this PR if you think this is a lot of work to be done now and we can open an issue for that :)